examples/bri-3/README.md

```diff
@@ -28,11 +28,17 @@ This reference implementation is being built from scratch and will contain a lim
 ## Installation
 
 ```bash
+# DB
+
 $ docker run --name postgres -e POSTGRES_PASSWORD=example -p 5432:5432 -d postgres # start a postgres container
 $ create a .env file based on the .env.sample # provide a connection string for the db instance
 $ npm install # install project dependencies
 $ npm run prisma:generate # generate the prisma client 
 $ npm run prisma:migrate:dev # migrate the db to latest state
+
+# Messaging
+
+$ docker run -p 4222:4222 -d nats 
```

Comment:
will there be a need for docker compose soon?

Reply:
Great question. We ll have to wait and see ;).